src/panels/profile/dialog-ha-mfa-module-setup-flow.ts

```diff
@@ -0,0 +1,290 @@
+              </div>`
+            : html`${this._step.type === "abort"
+                ? html` <ha-markdown
+                    allowsvg
+                    breaks
+                    .content=${this.hass.localize(
+                      `component.auth.mfa_setup.${this._step.handler}.abort.${this._step.reason}`
+                    )}
+                  ></ha-markdown>`
+                : this._step.type === "create_entry"
+                ? html`<p>
+                    ${this.hass.localize(
+                      "ui.panel.profile.mfa_setup.step_done",
+                      "step",
+                      this._step.title
+                    )}
+                  </p>`
+                : this._step.type === "form"
+                ? html` <ha-markdown
+                      allowsvg
+                      breaks
+                      .content=${this._computeStepDescription()}
+                    ></ha-markdown>
+                    <ha-form
+                      .data=${this._stepData}
+                      .schema=${this._step.data_schema}
+                      .error=${this._step.errors}
+                      .computeLabel=${this._computeLabel}
+                      .computeError=${this._computeError}
+                      @value-changed=${this._stepDataChanged}
+                    ></ha-form>`
+                : ""}`}
+        </div>
+        ${["abort", "create_entry"].includes(this._step?.type || "")
+          ? html`<mwc-button slot="primaryAction" @click=${this.closeDialog}
+              >${this.hass.localize(
+                "ui.panel.profile.mfa_setup.close"
+              )}</mwc-button
+            >`
+          : ""}
+        ${this._step?.type === "form"
+          ? html`<mwc-button
+              slot="primaryAction"
+              .disabled=${this._loading}
+              @click=${this._submitStep}
+              >${this.hass.localize(
+                "ui.panel.profile.mfa_setup.submit"
+              )}</mwc-button
+            >`
+          : ""}
+      </ha-dialog>
+    `;
+  }
+
+  static get styles(): CSSResult[] {
+    return [
+      haStyleDialog,
+      css`
+        .error {
+          color: red;
+        }
+        ha-dialog {
+          max-width: 500px;
+        }
+        ha-markdown {
+          --markdown-svg-background-color: white;
+          --markdown-svg-color: black;
+          display: block;
+          margin: 0 auto;
+        }
+        ha-markdown a {
+          color: var(--primary-color);
+        }
+        .init-spinner {
+          padding: 10px 100px 34px;
+          text-align: center;
+        }
+        .submit-spinner {
+          margin-right: 16px;
+        }
+      `,
+    ];
+  }
+
+  protected firstUpdated(changedProperties) {
+    super.firstUpdated(changedProperties);
+    this.hass.loadBackendTranslation("mfa_setup", "auth");
+    this.addEventListener("keypress", (ev) => {
+      if (ev.keyCode === 13) {
```

Comment:
Off topic: We use this "mystic" 13 now in multiple places, I wonder if it make sense to create a Keypress enum for this and others that can be reused.

Reply:
We should just use `ev.key === "Enter"`